torch/onnx/_patch_torch.py

```diff
@@ -8,9 +8,9 @@
 from torch._C import _onnx as _C_onnx
```

Comment:
Not moving this file to keep the patching behavior more obvious (easier to find)